packages/desktop-client/vite.config.mts

```diff
@@ -109,7 +109,7 @@ export default defineConfig(async ({ mode }) => {
     build: {
       target: 'es2022',
       sourcemap: true,
-      outDir: 'build',
+      outDir: mode === 'desktop' ? 'build-electron' : 'build',
```

Comment:
Needs to be seperate build folder so we don't break the server. The server points directly to the _build_ folder so we **can't** change it to _build/electron_ and _build/web_